### DISCUSSION TITLE
Weird Image rendering

### DISCUSSION THREAD

**Discussion by mongchanghsi** [vercel/next.js#54727] [Help]
### Summary

![image](https://github.com/vercel/next.js/assets/32842582/19437e97-e001-43f8-841c-84202f8dd8c4)
When I am using next/image, I often come across where my image render having a line on the top. It is as if bottom 1px of image is rendered on the top of the image instead. How do I solve this issue?

Typically how I code my image is
```
<div style={{ position: 'relative', width: '100px', height: '100px' }}>
    <Image src={someStaticImage} alt="Image Name" fill/>
</div>
```

I am using next 13.4.1

**Comment by devjiwonchoi** [comment]
Could you share a repro?

**Reply by mongchanghsi** [reply]
Repo: https://github.com/mongchanghsi/next_image_issue
Component file: https://github.com/mongchanghsi/next_image_issue/blob/master/src/views/Landing/index.tsx
Screenshot:
![image](https://github.com/vercel/next.js/assets/32842582/4e4b3069-8423-443b-bfe4-68956a908fbd)
Highlighted Screenshot:
![image](https://github.com/vercel/next.js/assets/32842582/cc044efb-a8e1-4975-9a00-325bdb533da6)

**Reply by devjiwonchoi** [reply]
Thanks, I'll take a look

**Reply by devjiwonchoi** [reply]
![Screenshot 2023-09-02 at 8 14 36 PM](https://github.com/vercel/next.js/assets/120007119/983ae31d-2ee5-4f22-b0f0-2ee852965bbd)

@mongchanghsi hey, I've failed to reproduce. Are you facing this issue on production also?

**Reply by mongchanghsi** [reply]
You can try zoom in a little, sometimes default zoom is fine sometimes is not. When you zoom in a little, the line will appear

**Reply by mongchanghsi** [reply]
I am facing this in some of my production apps.

Unable to share the url to one of my apps but here's a snippet
![image](https://github.com/vercel/next.js/assets/32842582/725d5ca9-6c7c-4d82-beee-bf9e4b415ab4)

**Reply by devjiwonchoi** [reply]
![Screenshot 2023-09-02 at 8 29 41 PM](https://github.com/vercel/next.js/assets/120007119/6d462614-bd08-4289-b4d9-97349af429c3)

Seems alright on close up also.

**Reply by devjiwonchoi** [reply]
Have you tested on other displays also?

**Reply by mongchanghsi** [reply]
Can update the height and width to 

```
  height: 168px;
  width: 121px;
```

Tested in default zoom, the line should appear
![image](https://github.com/vercel/next.js/assets/32842582/e38e9fec-e778-418c-868e-492e21035459)

**Reply by mongchanghsi** [reply]
![image](https://github.com/vercel/next.js/assets/32842582/f8129b82-c5ca-41b8-b455-8d367eccd1a9)
Line is visible in iOS Chrome as well

**Reply by devjiwonchoi** [reply]
@mongchanghsi  Yes I do see the line, I'll investigate. Thank you.

**Comment by devjiwonchoi** [comment]
Partial answer... for now.

Seems like it's a subpixel issue, which is a browser-related issue.

One suggestion is to try to use `objectFit='contain'`, and it partially resolves the issue.

Another suggestion is to redraw the SVG from base64 to proper vectors since this is the main reason for this issue.

**Reply by chronictanvi** [reply]
Tried this, doesn't work.
